features/zoeken-kadasternatuurlijkpersoon.feature

```diff
@@ -0,0 +1,56 @@
+Functionaliteit: zoeken kadaster natuurlijkpersoon
+
+    Achtergrond:
+        Gegeven de registratie kadasternatuurlijk personen kent de persoon: Willem Jansen, man, geboortedatum 01-01-1979
```

Comment:
Een tabel hier gebruiken is duidelijker (denk ik):

Gegeven de registratie kadasternatuurlijk personen kent de personen
| naam               | geslacht | geboortedatum |
| Willem Jansen | man    | 01-01-1979 |
| Sigrid Jansen   | vrouw | 20-02-1980 |
| Luuk Janssen   | man    | onbekend   |